heron/executor/src/python/heron_executor.py

```diff
@@ -33,10 +33,10 @@ def print_usage():
       " <instance_distribution> <zknode> <zkroot> <tmaster_binary> <stmgr_binary> "
       " <metricsmgr_classpath> <instance_jvm_opts_in_base64> <classpath> "
       " <master_port> <tmaster_controller_port> <tmaster_stats_port> <heron_internals_config_file> "
-      " <component_rammap> <component_jvm_opts_in_base64> <pkg_type> <topology_jar_file>"
+      " <component_rammap> <component_jvm_opts_in_base64> <pkg_type> <topology_bin_file>"
       " <heron_java_home> <shell-port> <heron_shell_binary> <metricsmgr_port>"
       " <cluster> <role> <environ> <instance_classpath> <metrics_sinks_config_file> "
-      " <scheduler_classpath> <scheduler_port>")
+      " <scheduler_classpath> <scheduler_port> <python_instance_binary>")
 
 def do_print(statement):
   timestr = datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
I think it is a good idea to use logging uniformly - what is rationale for using do print?